### PR TITLE
Quiz app refactored using ES6 syntax and build tools.

### PR DESCRIPTION
**Pull request by rlynn523**

### CODE REVIEW

File: build/index.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+	<section class="question-answer">
+		<div class="question-title">
+			<h2><strong>Question #<span class="counter">1</span></strong></h2>
+			<p></p>
+		</div>
+		<form class="quiz-form">
+			<div class="answers"></div>
+			<div class="new-game">New Game!</div>
+		</form>
+		<br>
```
Comment (aric87) [inline build/index.html]: Why the double break? If it's for space, use CSS

File: build/normalize.css [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+*:after {
+  box-sizing: inherit;
+}
+body {
+  background-color: #FFFFFF;
+  background-repeat: no-repeat;
+  background-image: url("http://www.thatericalper.com/wp-content/uploads/2014/04/Jerryseinfeld.jpg");
+  background-size: 45%;
+  background-attachment: fixed;
+  font-family: 'Roboto Slab', serif;
```
Comment (aric87) [inline build/normalize.css:15]: background uses double quotes, font uses single. Pick one, and be consistent throughout the file

File: css/index.less [hunk 3]
```diff
@@ -1,81 +1,105 @@
...
+
+@roboto: 'Roboto Slab', serif;
+@white: #FFFFFF;
+@black: #000000;
+@red: #E30922;
+@yellow: #FCD920;
+
 body {
-	background-color: #FFFFFF;
+  background-color: @white;
```
Comment (aric87) [inline css/index.less]: should be indented the same as everything else

File: css/index.less [hunk 4]
```diff
@@ -1,81 +1,105 @@
...
 	text-align: center;
 	border-radius: 10px;
 	padding-top: 8px;
-	border-color: #E30922;
+	border-color: @red;
 	border-style: solid;
 }
 .submit:hover, .new-game:hover {
 	cursor: pointer;
+  transition: background-color 0.5s ease;
```
Comment (aric87) [inline css/index.less]: indentation

File: css/index.less [hunk 5]
```diff
@@ -108,17 +136,14 @@ input {
...
 @media only screen and (max-width: 1050px) {
 	.question-title {
-		height: 4em;
-		width: 25em;
-	}
-	.answer {
-		height: 4em;
-		width: 20em ;
+		p {
+			font-size: 15px;
```
Comment (aric87) [inline css/index.less]: you should be consistent in the units you use. You have px here, and 3 lines down, you're using ems....

File: build/index.html (hunk 1 above)
Comment (rlynn523) [inline build/index.html]: added margin-bottom

**Pull request merged**